CppCoreGuidelines.md

```diff
@@ -2663,10 +2681,14 @@ A `span` represents a range of elements, but how do we manipulate elements of th
 
     void f(span<int> s)
     {
-        for (int x : s) cout << x << '\n';  // range traversal (guaranteed correct)
-        for (int i = 0; i < s.size(); ++i) cout << x << '\n';  // C-style traversal (potentially checked)
-        s[7] = 9;                           // random access (potentially checked)
-        std::sort(&s[0], &s[s.size() / 2]);    // extract pointers (potentially checked)
+        // range traversal (guaranteed correct)
+        for (int x : s) cout << x << '\n';
```

Comment:
I think for sequences of commented lines, some extra white space might be good, i.e.:

    // range traversal (guaranteed correct)
    for (int x : s) cout << x << '\n';
    
    // C-style traversal (potentially checked)
    for (int i = 0; i < s.size(); ++i) cout << x << '\n';

It makes it more clear which lines the comments are paired with.